pages/Basic-Machines.md

```diff
@@ -0,0 +1,24 @@
+# Basic Machines
```

Comment:
Wiki pages should not include headers.
They have headers automatically, this would create two.